src/ui/storage/index.ts

```diff
@@ -136,6 +141,27 @@ export default class ApplicationStorage implements IApplicationStorage {
     return value;
   }
 
+  async updateBindingPhraseHistory(bindingPhrase: string): Promise<void> {
+    let history = await this.getBindingPhraseHistory();
+    history = history.filter((item) => item !== bindingPhrase);
+    history.push(bindingPhrase);
+    history.length = Math.min(history.length, 10);
+    localStorage.setItem(BINDING_PHRASE_HISTORY_KEY, JSON.stringify(history));
+  }
+
+  async getBindingPhraseHistory(): Promise<string[]> {
+    const history = localStorage.getItem(BINDING_PHRASE_HISTORY_KEY);
+    if (history === null) {
+      return [''];
```

Comment:
It would be better to return an empty array when there are no history items rather than an array with 1 entry that is an empty string.